test/unit/math/mix/scal/fun/grad_2F1_test.cpp

```diff
@@ -14,9 +14,9 @@ TEST(ProbInternalMath, grad2F1_fd1) {
   fvar<double> gradC;
   stan::math::grad_2F1(gradA,gradC,a, b, c, z);
 
-  EXPECT_NEAR(0.4617734315397201318453321291834046302225919173588625242,gradA.val_,1e-6);
-  EXPECT_NEAR(0.163714876516383746459968120418298168600425943651588679302872,gradA.d_,1e-5);
-  EXPECT_NEAR(-0.461773435230326182245722531773361592054302268779753796048,gradC.val_,1e-6);
+  EXPECT_NEAR(0.4617734315397201318453321291834046302225919173588625242,gradA.val_,1e-8);
```

Comment:
We only use double precision (so at most 16 digits), and then you test to only 1e-8, so these long values are distracting and overkill.